cuda_core/examples/simple_multi_gpu_example.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+from cuda.core.experimental import Device
+from cuda import cuda, cudart, nvrtc
```

Comment:
This is unused (and will be removed once you run pre-commit as Vlad suggested above IIRC).